DESCRIPTION

```diff
@@ -1,11 +1,11 @@
 Package: drat
 Type: Package
 Title: 'Drat' R Archive Template
-Version: 0.1.5
-Date: 2019-03-28
+Version: 0.1.6
```

Comment:
It's minor but I tend to set those as these "final" numbers only when I prepare the release, so I would have called this 0.1.5.1 instead.

Reply:
Changed this to 0.1.5.1. I am quite keen to get it to CRAN, since I tripped over this this morning. It would save me some time by not having to prepare a preliminary solution

Reply:
I can probably help with that especially if you can help with testing (i.e. I have no macOS binaries hanging around)

Reply:
Sure thing. I did all the testing on macOS `Catalina` and thats how I noticed the problem. The code in the PR works for R `3.6` and `4.0`. I inserted packages, archived and pruned.

Do you have other test in mind?

Reply:
The macOS case is, I think,  by far the most complicated.

Reply:
@eddelbuettel I took a quick look at the proposed changes and agree they seem to improve the state of things. The original function was intended to address problems we encountered with MacOS, and it seems we missed some key functionality. So I'm all in support of fixing these issues while retaining the ability to install and upgrade mac packages from the CRAN-sanctioned locations. Thanks for reaching out.